packages/x-date-pickers/src/DesktopDateTimePicker/DesktopDateTimePicker.types.ts

```diff
@@ -26,6 +26,11 @@ export interface DesktopDateTimePickerSlotsComponentsProps<TDate>
 export interface DesktopDateTimePickerProps<TDate>
   extends BaseDateTimePickerProps<TDate>,
     DesktopOnlyPickerProps<TDate> {
+  /**
+   * Display ampm controls under the clock (instead of in the toolbar).
+   * @default true
```

Comment:
Are you sure it's worth going with these overrides trying to provide a more correct default? 🤔 
It's nice, I agree, but in that case, we'd need to fix an issue with `DateTimePicker`, maybe omit `ampmInClock` when creating `DateTimePickerProps`?
Or just go without overriding the default and show `true on desktop, false on mobile` 🙈 
What's your take on it @flaviendelangle?

Reply:
I did not realize it would also affect all the other component. I moved back to `true on desktop, false on mobile`  for every one to avoid side effects